jgt_tools/data/defaults.csv

```diff
@@ -1,6 +1,7 @@
 env_setup_commands,poetry run pip install --upgrade pip<19
 env_setup_commands,poetry install
 env_setup_commands,poetry run pre-commit install
+env_setup_commands,poetry run pre-commit autoupdate,in case of a pre-exsiting pre-commit environment that is stale
```

Comment:
So looking at the [documentation](https://pre-commit.com/#pre-commit-autoupdate) for this call, it looks like it is going to bump any defined hooks up to the latest version, disregarding the version specified in the pre-commit config file. Are we sure that's what we want?

It looks like there is [another command](https://pre-commit.com/#pre-commit-install-hooks) to force a hook install of the specified version, so we could either make this line `poetry run pre-commit install-hooks` or add an `--install-hooks` flag to the line above it.

Reply:
Yeah, I wasn't sure about which one to use (DNM was double duty).
Unfortunately I don't have any way to rewind my pre-commit venv to see if the `install-hooks` would have fixed the problem or not.
I'm open to whichever command the group thinks is the right thing.

Reply:
Well, I ran in to this again, just now, when coming back to my sphinx-gherkindoc repo on a different laptop. `poetry run pre-commit install-hooks` and the `--install-hooks` flags both failed to fix the issue.

Reply:
i had to use the `poetry run pre-commit autoupdate` to get black and flake8 working again.

Reply:
Having run `autoupdate`, the `.pre-commit-config.yaml` file changed.
Reverting that change, things borked again.
Running `poetry run pre-commit clean` and the run `./self-check.py` got things working again.
So maybe that should be the direction to go?
(Also, being somewhat grumpy about "oh great, let me try working on the code on _this_ computer now, oh wait, no, pre-commit sabotaged me again" I'm not really remembering all the wonderful things it is providing, just the hassles...)